packages/core/src/Stores/client-store.js

```diff
@@ -658,6 +658,14 @@ export default class ClientStore extends BaseStore {
         return this.isDxtradeAllowed(this.landing_companies);
     }
 
+    @computed
+    get is_dbot_allowed() {
```

Comment:
you can use this function in app_store `showDigitalOptionsMaltainvestError` to show error